docs/en/trust.rst

```diff
@@ -5,12 +5,12 @@
 The Infrastructure of Trust
 +++++++++++++++++++++++++++
 
-The EUDI Wallet Architecture Reference Framework (`EIDAS-ARF`_) defines the Trust Model as *"Collection of rules that ensure the legitimacy of the components and the entities involved in the EUDI Wallet ecosystem."*.
+The EUDI Wallet Architecture Reference Framework (`EIDAS-ARF`_) defines the Trust Model as a *"collection of rules that ensure the legitimacy of the components and the entities involved in the EUDI Wallet ecosystem."*.
 
-In this section is defined how the Trust Model is implemented in a infrastructure of Trust based on
-OpenID Connect Federation 1.0 `OIDC-FED`_, where its Federation API, is used for the distribution of metadata, raw public keys, metadata policies, X.509 certificates and their revocation status.
+In this section, we define how the Trust Model is implemented in an infrastructure of Trust based on
```

Comment:
I love your revision, however I would not use "we" here

Reply:
@peppelinux what would you use? is the following suggestion better?
```suggestion
This section defines how the Trust Model is implemented in an infrastructure of Trust based on
```